tests/authAPI.spec.js

```diff
@@ -0,0 +1,119 @@
+import { test, expect } from "@playwright/test";
+import { generateUserCredentials, ERRORS } from "../fixtures";
+import { RegisterAPI } from "../pom/modules/api/registerAPI";
+import { LoginAPI } from "../pom/modules/api/loginAPI";
+
+let loginAPI;
+let registerAPI;
+const { username, email, pass, registerdUser, registerdEmail ,invalidEmail} =
+  generateUserCredentials(5);
+let loginEmail = email;
+let loginPassword = pass;
+
+test.describe.configure({ mode: "serial" });
+
+test.describe("Registration API tests", async () => {
+  test.beforeEach("start pom", async ({ page }) => {
```

Comment:
Milsim da si mogao samo u `beforeAll` jednom da instanciras klasu pa da je reuse-ujes, isto vazi i za login.

Reply:
Pokusao sam sada beforeAll i ako uradim isto kao i u beforeEach dobijam gresku:
Error: "context" and "page" fixtures are not supported in "beforeAll" since they are created on a per-test basis

Pokusao sam i to da resim, ali nisam uspeo.